### PR TITLE
Finish the Romanian translation

### PR DESCRIPTION
**Pull request by mihaimaruseac**
As per #472

### CODE REVIEW

File: README-ro.md [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+
+- `ldd`: informații despre bibliotecile dinamice
+
+- `nm`: simboluri din fișierele obiect
+
+- `ab`: benchmarking pentru servere web
+
+- `strace`: depanare apeluri de sistem
+
+- [`mtr`](http://www.bitwizard.nl/mtr/): alternative mai bună pentru `traceroute` pentru depanarea rețelei
```
Comment (bgdnlp) [inline README-ro.md]: Typo, "alternativă mai bună la" instead of "alternative mai bună pentru"

File: README-ro.md [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+
+- `vmstat`: statistici despre folosirea memoriei
+
+- `htop`: versiune îmbunătățită a `top`
+
+- `last`: istoria login-urilor
+
+- `w`: cine este logged-in
+
+- `id`: informații despre useri/grupuri
```
Comment (bgdnlp) [inline README-ro.md]: "utilizatori" instead of "useri"

File: README-ro.md [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+- `lsmod` și `modinfo`: afișează module de kernel
+
+- `fortune`, `ddate`, și `sl`: depinde dacă considerați locomotive și citate "utile" (utilitare distractive)
+
+
+## Doar pentru OS X
+
+Aceste sfaturi sunt relevante *doar* pentru OS X.
+
+- Managementul pachetelor cu `brew` (Homebrew) și/sau `port` (MacPorts). Aceste comenzi pot fi folosit pentru a instala multe din comenzile de mai sus pe un Mac.
```
Comment (bgdnlp) [inline README-ro.md]: Typo, "folosite" instead of "folosit"

File: README-ro.md [hunk 4]
```diff
@@ -0,0 +1,596 @@
...
+
+- Folosiți `mintty` ca fereastra de comenzi.
+
+- Accesați clipboard-ul Windows prin intermediul `dev/clipboard`.
+
+- Folosiți `cygstart` pentru a deschide un fișier arbitrar cu aplicația corespunzătoare
+
+- Accesați registrele Windows cu `regtool`.
+
+- Observați că o cale de tip `C:\` în Windows devine `/cygdrive/c` sub Cygwin, și că root-ul Cygwin, `/`, apare ca `C:\cygwin` pe Windows. convertiți între căile Cygwin și Windows cu `cygpath`. Acest lucru este util în scripturile care apelează programe Windows.
```
Comment (bgdnlp) [inline README-ro.md]: Typo, "Convertiți" instead of "convertiți"

File: README-ro.md [hunk 5]
```diff
@@ -0,0 +1,596 @@
...
+
+- Puteți executa majoritatea task-urilor de administrare a sistemului Windows din linia de comandă prin învățarea și folosirea `wmic`.
+
+- O altă opțiune de a obtine un sistem similar Unix sub Windows este [Cash](https://github.com/dthree/cash). Țineți cont că doar câte utilitare Unix sunt în prezent disponibile în acest mediu.
```
Comment (bgdnlp) [inline README-ro.md]: Typo, "obține" instead of "obtine"

**Review by bgdnlp — COMMENTED** [review]
First, I must say thank you, pleasantly surprised to see a good translation. Most of the corrections are typos and things that are too english-like. I'm sure there's a bunch I missed and some that can be argued, but I think that it is very good for what it is and very helpful. Thank you.

**Comment by mihaimaruseac** [comment]
Thanks for the suggestions and the feedback. I made all the changes (except the one about "parametri").

File: README-ro.md [hunk 6]
```diff
@@ -0,0 +1,596 @@
...
+
+## Procesarea fișierelor și a datelor
+
+- Pentru a localiza un fișier după nume, în directorul curend, `find . -iname '*something*'` (sau similar). Pentru a găsi un fișier oriunde, `locate something` (dar țineți cont de faptul că `updatedb` s-ar putea să nu fi indexat fișierele recente).
+
+- Pentru a căuta în interiorul fișierelor (mai avansat decât `grep -r`), folosiți [`ag`](https://github.com/ggreer/the_silver_searcher).
+
+- Pentru a converti HTML la text: `lynx -dump -stdin`
+
+- For Markdown, HTML, and all kinds of document conversion, try [`pandoc`](http://pandoc.org/).
```
Comment (mihaimaruseac) [inline README-ro.md]: Good catch, missed it :(

File: README-ro.md [hunk 7]
```diff
@@ -0,0 +1,596 @@
...
+```
+
+- În scripturile Bash, subshell-urile (delimitate de paranteze) sunt un mod convenabil pentru a grupa comenzi. De exemplu, va puteți muta într-un alt director:
+```bash
+      # do something in current dir
+      (cd /some/other/dir && other-command)
+      # continue in original dir
+```
+
+- În Bash, există mai multe variante de expandare a variabilelor. Verifacți dacă o variabilă există: `${name:?error message}`. De exemplu, dacă un script necesită un argument unic, folosiți `input_file=${1:?usage: $0 input_file}`. Folosți o valoare implicită pentru cazul când o variabilă este goală: `${name:-default}`. Pentru parametri opționali în exemplul anterior, folosiți `output_file=${2:-logfile}`. Dacă `$2` este omis (gol), `output_file` va fi setat la `logfile`. Expandare aritmetică: `i=$(( (i + 1) % 5 ))`. Secvențe: `{1..10}`. Tăierea unor secvențe din șiruri: `${var%suffix}` și `${var#prefix}`. De exemplu, dacă `var=foo.pdf`, atunci `echo ${var%.pdf}.txt` va scrie `foo.txt`.
```
Comment (mihaimaruseac) [inline README-ro.md]: I consulted DOOM here and I'd still go with `parametri`.

**Comment by jlevy** [comment]
👍 Awesome. Thank you @mihaimaruseac and @bgdnlp for the careful review.

@mihaimaruseac you're now Romanian maintainer, and @bgdnlp I'll credit you as a translator as well. I'll update the credits.

**Pull request merged**